src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/OpenApiParserTests.cs

```diff
@@ -1374,6 +1374,49 @@ public async Task Teams_PostCardAndWaitForResponse()
 
             Assert.NotNull(functionsWithWebhooks.FirstOrDefault(f => f.Name == "PostCardAndWaitForResponse"));
             Assert.Null(functionsWithoutWebhooks.FirstOrDefault(f => f.Name == "PostCardAndWaitForResponse"));
+        }
+
+        [Fact]
+        public async Task Teams_GetMessageLocation_WithSuggestions()
+        {
+            using var testConnector = new LoggingTestServer(@"Swagger\Teams.json", _output);
+            using var httpClient = new HttpClient(testConnector);
+            using PowerPlatformConnectorClient client = new PowerPlatformConnectorClient("https://tip1002-002.azure-apihub.net", "7592282b-e371-e3f6-8e04-e8f23e64227c" /* environment Id */, "shared-cardsforpower-eafc4fa0-c560-4eba-a5b2-3e1ebc63193a" /* connectionId */, () => "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dC...", httpClient) { SessionId = "a41bd03b-6c3c-4509-a844-e8c51b61f878" };
+
+            BaseRuntimeConnectorContext runtimeContext = new TestConnectorRuntimeContext("DV", client, console: _output);
+
+            ConnectorFunction[] functions = OpenApiParser.GetFunctions(new ConnectorSettings("DV") { Compatibility = ConnectorCompatibility.SwaggerCompatibility }, testConnector._apiDocument).ToArray();

```

Comment:
>ConnectorSettings [](http://example.com/codeflow?start=75&length=17)

This is where I'd set the new flag
yes, that will be connector wide but i think that's acceptable as long as it doesn't impact other connectors (other than Teams)

Reply:
yes, updated